### DISCUSSION TITLE
Have I made the package.json file correctly?

### DISCUSSION THREAD

**Discussion by AdamVasarhelyi2** [jspsych/jsPsych#3084] [Q&A] [answered]
Hi, I've been working on my first extension for JsPsych. To make the package.json filem, I navigated to the extension directory in cmd.exe, entered 'npm init' and filled out the prompts to make a package.json file. 

_Have I made the package.json file correctly?_

Thanks for any help!!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @AdamVasarhelyi2 

This is a normal way to make a package.json file. 

If you'd like one that is already configured for extension development, you can start with our [template for JavaScript](https://github.com/jspsych/jspsych-contrib/blob/main/packages/extension-template) or [template for TypeScript](https://github.com/jspsych/jspsych-contrib/tree/main/packages/extension-template-ts).

**Reply by AdamVasarhelyi2** [reply]
Thank you so much! I just wanted to be sure I did it correctly :)
